### ISSUE TITLE
tempsense: avoid metal 1 in the VIN route

### ISSUE THREAD

**Issue by luccareinehr** [idea-fasoc/OpenFASOC#132]
Lower metal layers have higher resistivity, so we should ignore them for power routing

**Comment by msaligane** [comment]
Lower metal layer should not be used for routing these special nets. We also have to find a way to avoid single via connection.
We really need a Special router or point-to-point router.

**Comment by luccareinehr** [comment]
We can set the top and bottom routing layers with the detailed_place command

So we can first route all nets except r_VIN, then just route r_VIN limiting the layers

**Comment by msaligane** [comment]
Yes, that should work. But at this point, we have too many hacks to do something so simple.
We need to create a specification so that OpenROAD developers add it.

**Comment by saicharan0112** [comment]
@luccareinehr any plans to work on this?

**Comment by luccareinehr** [comment]
@saicharan0112 not really, sorry... I remember all I could try were some "hacks" with openroad commands, which eventually became too complicated. Hence the need for the special router Mehdi mentions

**Comment by saicharan0112** [comment]
No problem! Let's see if anyone could pick this up.
